schema.yaml

```diff
@@ -143,6 +158,9 @@ exports:
       - lang: go
         source: |
           return NoInputNoOutputHost()
+      - lang: rust
+        source: |
+          unsafe { pdk::noInputNoOutputHost() }
```

Comment:
for each of these, do you think we could add a "safe" wrapper which is what we'd expect the end user to call? 

if we do have a safe wrapper, can we use that in this so that the test calls the wrapper instead?

Reply:
Good idea - I just updated xtp-rust-bindgen to improve the handling of imports, and updated the schema in this PR